Doc/Makefile

```diff
@@ -122,9 +123,11 @@ clean:
 	-rm -rf build/* $(VENVDIR)/*
 
 venv:
-	$(PYTHON) -m venv $(VENVDIR)
-	$(VENVDIR)/bin/python3 -m pip install -U Sphinx blurb
-	@echo "The venv has been created in the $(VENVDIR) directory"
+	@if [ "$(SPHINXBUILD)" == "$(VENVDIR)/bin/sphinx-build" ]; then \
```

Comment:
This should also check whether `$(SPHINXBUILD) --version > /dev/null 2>&1` works, and only build the venv if it doesn't.

Reply:
I fixed your other comments. The tests pass. This, about checking whether the command runs, is a bit more interesting. Since the `venv` Makefile action will only run if the venv dir is absent (is that right?), it means that `$(SPHINXBUILD)` could never work, until the venv is created.  So to get into the body, we need the situation to be that the venv has just been created. Therefore, (in this situation), `$(SPHINXBUILD)` could never work. So why check for it?  (fyi all my experience with makefiles has been on this issue; feel free to assume that know very little about how it works :). 

The other, separate complication is that `blurb` now uses the same venv so we have to be careful to not mess up anything to do with that other tool.  I've run `make html` several times. It looks ok right now, and the `blurb` action appears to work correctly.